src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,94 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private T[] array;
+    private int size = 0;
+
+    public ArrayList() {
+        array = (T[]) new Object[DEFAULT_SIZE];
+    }
+
+    private void grow() {
+        T[] arrayCopy = (T[]) new Object[array.length * 3 / 2];
```

Comment:
let's make constant GROV_COEFFICIENT = 1.5 and use it to create array with new capacity